recipes/pbzip2/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.1.13" %}
+{% set major_version = "1.1" %}
+{% set sha1 = "f61e65a7616a3492815d18689c202d0685fe167d" %}
+{% set md5 = "4cb87da2dba05540afce162f34b3a9a6" %}
+
+package:
+  name: pbzip2
+  version: {{ version }}
+
+source:
+  fn: pbzip2-{{ version }}.tar.gz
+  url: https://launchpad.net/pbzip2/{{ major_version }}/{{ version }}/+download/pbzip2-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - bzip2
```

Comment:
So this is a bit tedious (sorry). The pinning bot will try to do this anyways even though we are statically linked. Maybe we should fix the bot? IDK that is a different discussion though. Thus I'm going to recommend we do this anyways. Fortunately it is an easy change.

Please change `bzip2` -> `bzip2 1.0.*`.